xio-tls/src/main/java/com/xjeffrose/xio/tls/X509Certificate.java

```diff
@@ -14,36 +14,19 @@
  * limitations under the License.
```

Comment:
I don't think anything is using this class? If so we can remove it and get rid of some javac warnings about touching `sun.security` classes.